ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/parameters/DepositParameter.java

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright contributors to Hyperledger Besu
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.hyperledger.besu.ethereum.api.jsonrpc.internal.parameters;
+
+import org.hyperledger.besu.datatypes.BLSPublicKey;
+import org.hyperledger.besu.datatypes.BLSSignature;
+import org.hyperledger.besu.datatypes.DepositWithdrawalCredential;
+import org.hyperledger.besu.datatypes.GWei;
+import org.hyperledger.besu.ethereum.core.Deposit;
+
+import java.util.Objects;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonGetter;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.vertx.core.json.JsonObject;
+import org.apache.tuweni.units.bigints.UInt64;
+
+public class DepositParameter {
+
+  private final String publicKey;
+
+  private final String withdrawalCredentials;
+  private final String amount;
+
+  private final String signature;
+  private final String index;
+
+  @JsonCreator
+  public DepositParameter(
+      @JsonProperty("pubKey") final String pubKey,
+      @JsonProperty("withdrawalCredentials") final String withdrawalCredentials,
```

Comment:
In the spec for [DepostiV1]( https://github.com/mkalinin/execution-apis/blob/eip6110/src/engine/experimental/eip6110.md#depositv1) the field is named "withdrawal_credentials"